src/shared/components/Select/Select.tsx

```diff
@@ -1,42 +1,57 @@
-import React, { forwardRef } from 'react'
+import React from 'react'
```

Comment:
@drillprop Please take a look to make sure I didn't break anything

Reply:
Tested on playgrounds with changed values and on the storybook. Looks good for me. I like this change a lot 👍🏻